From 439f0bea1f672cbb2e395e493d8d5b8a86aa6e6f Mon Sep 17 00:00:00 2001
From: Jo Shields <jo.shields@xamarin.com>
Date: Mon, 6 Jun 2016 16:57:24 +0100
Subject: [PATCH] Move call to sn.exe to *after* cil-stringreplacer.exe

This makes sure mscorlib.dll still has a valid signature (fixes nightly builds on Linux)
---
 mcs/build/library.make | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mcs/build/library.make b/mcs/build/library.make
index c8379b2..f9b197c 100644
--- a/mcs/build/library.make
+++ b/mcs/build/library.make
@@ -277,10 +277,10 @@ $(the_lib): $(the_libdir)/.stamp
 
 $(build_lib): $(response) $(sn) $(BUILT_SOURCES) $(build_libdir:=/.stamp)
 	$(LIBRARY_COMPILE) $(LIBRARY_FLAGS) $(LIB_MCS_FLAGS) -target:library -out:$@ $(BUILT_SOURCES_cmdline) @$(response)
-	$(Q) $(SN) -R $@ $(LIBRARY_SNK)
 ifdef RESOURCE_STRINGS_FILES
 	$(Q) $(STRING_REPLACER) $(RESOURCE_STRINGS_FILES) $@
 endif
+	$(Q) $(SN) -R $@ $(LIBRARY_SNK)
 
 ifdef LIBRARY_USE_INTERMEDIATE_FILE
 $(the_lib): $(build_lib)
-- 
2.7.4

